integration_tests/plugins/test_rust_plugin.py

```diff
@@ -69,13 +69,27 @@ def test_stage_rust_plugin_with_conditional_feature(self):
 
     def test_stage_rust_with_source_subdir(self):
         self.run_snapcraft('stage', 'rust-subdir')
-
+        
```

Comment:
@roxyd note in here that you replaced an empty line with a line that has spaces.
Our automated static checks will fail here in travis. Please remove the spaces, and leave just the empty line.